hypervisor/include/arch/x86/trusty.h

```diff
@@ -115,6 +134,11 @@ struct trusty_startup_param {
 	uint8_t padding[4];
 };
 
+#ifdef WORKAROUND_FOR_TRUSTY_4G_MEM
+void create_secure_world_ept(struct vm *vm, uint64_t gpa_orig,
```

Comment:
move these 2 lines to line 97, so that all you changes stay together. This way, we can reduce the revert conflict in future